### DISCUSSION TITLE
IBM iX showcase for your client Barmer

### DISCUSSION THREAD

**Discussion by UpperCod** [atomicojs/atomico#92] [General]
**Context**: The IBM iX team has created a really beautiful design system for their client Barmer and for the web version of their design system they have used Atomico.

![preview](https://user-images.githubusercontent.com/9159849/188505279-c570a7ec-b8e7-4dc2-9413-992a930040d1.png)

We invite you to share your questions and meet the IBM iX team who have used Atomico for their client Barmer.

**IBM iX Team**: @efoken and @spielersun

**News**
- [IBM iX was awarded with a Red Dot in the Brands & Communication Design discipline for the BARMER mobile experience.](https://ibmix.de/en/blog/red-dot-design-award-for-barmer/)

**Comment by efoken** [comment]
@UpperCod you can add @spielersun in the header.

Ah and there's a typo in the subject "IBM iX" not "IBX iX" 😉

EDIT: Note that at the moment we cannot make the Storybook and/or documentation websites public. Please let's keep it secret for now, but of course screenshots are fine 🙂

**Comment by UpperCod** [comment]
### **aecea**:  What were the crucial reasons to pick Atomico over other solutions (StencilJS, SolidJS, ...)? (Q&A from Discord)


> **efoken**:  This is simple: SolidJS is too different from React and StencilJS do not support hooks. We have a lot of React developers and wanted to make component development as simple as possible, and we already had about 50 React components before we started with the WebComponents
> We tried Haunted first but it did not work great and even Atomico was way more actively maintained 😉
In the beginning many WebComponents of the Barmer Design System were just copies of the React components

**Comment by UpperCod** [comment]
## aecea: Is the Barmer mobile app gonna using the planned React Native+Atomico integration? (Q&A from Discord)

> **efoken**: No this is not planned, as the Barmer mobile app is built using our React Native components and some screens are built with native code.
We have a huge monorepo and share lots of code between React Native (Web) and Atomico. For example we have shared hooks, utils and all component styles are in their own package and we are generating all CSS for WebComponents from React Native styles.
Many React Native components are very complex, but I really hope that we are able to use the Atomico RN integration someday, but currently I'm not sure if this is possible 😅
